contracts/cw1-subkeys/src/multitest/tests.rs

```diff
@@ -31,18 +30,20 @@ macro_rules! assert_sorted_eq {
 
 #[test]
 fn get_contract_version_works() {
-    let mut app = App::default();
+    let app = App::default();
 
-    let owner = Addr::unchecked("owner");
+    let owner = "owner";
 
-    let code_id = Cw1SubkeysCodeId::store_code(&mut app);
+    let code_id = Cw1SubkeysContractCodeId::store_code(&app);
```

Comment:
I am not a fan of the name `Cw1SubkeysContractCodeId` - why there is this `Conract` generated in the name? It it loooong.